nsis/src/nsis/NsisLoader.java

```diff
@@ -269,4 +275,40 @@ private void initPagesSection(InputStream is, Address startingAddr, Program prog
 			startingAddr = startingAddr.add(NsisPage.getPageSize());
 		}
 	}
+
+	/**
+	 * Initializes the section headers section and adds the it to the "program
+	 * Trees" view in Ghidra.
+	 * 
+	 * @param is
+	 * @param startingAddr
+	 * @param program
+	 * @param monitor
+	 * @throws LockException
+	 * @throws MemoryConflictException
+	 * @throws AddressOverflowException
+	 * @throws CancelledException
+	 * @throws DuplicateNameException
+	 * @throws CodeUnitInsertionException
+	 * @throws InvalidNameException
+	 */
+	private void initSectionHeaders(InputStream is, Address startingAddr, Program program,
+			TaskMonitor monitor, int nbEntries) throws LockException, MemoryConflictException,
+			AddressOverflowException, CancelledException, DuplicateNameException,
+			CodeUnitInsertionException, InvalidNameException {
+
+		String blockName = ".section_headers";
+		boolean readPermission = true;
+		boolean writePermission = false;
+		boolean executePermission = false;
+		createGhidraMemoryBlock(is, startingAddr, program, monitor,
+				NsisSection.getSectionSize() * nbEntries, blockName, readPermission,
+				writePermission, executePermission);
+
+		for (int i = 0; i < nbEntries; i++) {
+			NsisSection.STRUCTURE.setName("Section #" + (i + 1));
```

Comment:
You don't like counting from 0? ;-)

Or do the NSIS docs/code refer to the sections from 1?

Reply:
I think I don't like counting from 0 lol
Should I start at 0 or at 1?

Reply:
the NSIS docs don't label the sections with a number, I just thought it would look nicer that way in Ghidra

Reply:
It's probably ok to start from 1 if we are consistent.